prep-css-box-model/style.css

```diff
@@ -0,0 +1,10 @@
+body {
+  text-align: center;
+}
+
+button {
+  background-color: red;
+  color: white;
+
+  /* 👇 YOUR CODE HERE 👇 */
+}
```

Comment:
Hi Fredrick, for this exercise, please write your CSS code in the style.css file.